### DISCUSSION TITLE
Drag'n'drop support for file and media attachments

### DISCUSSION THREAD

**Discussion by x-iso** [bkaankose/Wino-Mail#160] [Ideas]
Currently it's impossible to drag'n'drop files or anything into Wino mail to attach them to the mail, that's quite a bummer. Ideally having different options for hovering over subject bar or body of the mail, so you can choose to just attach file or insert picture beside text

**Comment by Tiktack** [comment]
I believe it can be closed :)

**Reply by x-iso** [reply]
what do you mean? I don't see it being implemented

**Reply by Tiktack** [reply]
![image](https://github.com/bkaankose/Wino-Mail/assets/22834937/8fd3147c-7e99-4be4-995b-484ae6a6db2c)

**Comment by bkaankose** [comment]
Drag & drop is already implemented for files and images for composer.

Closing as fixed.

**Comment by x-iso** [comment]
I'm not sure it ever worked for me, but it definitely doesn't work in new v2 app
